src/components/Card/Card.tsx

```diff
@@ -21,16 +22,20 @@ export const Card: React.FunctionComponent<CardProps> = (props) => {
   );
 };
 
-const style = (theme: Theme) =>
-  stylesheet({
+const style = (theme: Theme) => {
+  return stylesheet({
     container: {
       boxShadow: shadow('2X', theme),
       background: theme.background,
       boxSizing: 'border-box',
       padding: '16px',
     },
+    elevation: {},
```

Comment:
do we need this?

Reply:
Nope, removed. Also-- what about the type definition for `elevations`-- I've added  the type in `helpers/elevations.ts`, but not as an `interface`.

Reply:
`type` is good enough. We don need `interface`.